templates/saphana.tpl.yaml

```diff
@@ -0,0 +1,183 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: {{ os }}-{{ item.workload }}-{{ item.flavor }}
+  annotations:
+    openshift.io/display-name: "Red Hat Enterprise Linux 8.4 Beta VM for SAP HANA workloads"
```

Comment:
Does the word `Beta` here has meaning rhel 8.4 beta or beta sap template?

Reply:
It is just a copy-pase error. :+1:

Reply:
Fixed.